raphtory/build.rs

```diff
@@ -3,10 +3,19 @@ use std::io::Result;
 fn main() -> Result<()> {
     prost_build::compile_protos(&["src/serialise/graph.proto"], &["src/serialise"])?;
     println!("cargo::rerun-if-changed=src/serialise/graph.proto");
+
+    println!("cargo::rustc-check-cfg=cfg(has_debug_symbols)");
+    if let Ok("true") = std::env::var("DEBUG").as_deref() {
```

Comment:
Have a look at the docs for the DEBUG flag, there are more options than true, you also want to turn this on if it is set to 2 I think.